packages/geant4/package.py

```diff
@@ -11,3 +11,7 @@ class Geant4(BuiltinGeant4):
     version("11.1.1.east", git="https://github.com/eic/geant4", branch="east-v11.1.1")
 
     patch("G4LogicalSkinSurface.patch", when="@11.2")
+
+    ## Fix commands in /particle/property/decay/
+    patch("https://github.com/Geant4/geant4/commit/e32bc92498d87fb42829b08348d7fad89bc89404.diff",
+          sha256="0581bd4869ab1a7459b639b63a041d01b672247e701f1106580bfce9209c47ed")
```

Comment:
```suggestion
    patch("https://github.com/Geant4/geant4/commit/e32bc92498d87fb42829b08348d7fad89bc89404.diff?full_index=1",
          sha256="a63b098f3214eaf4b5f34b3e434603b1cdbd13c919456782e933cd7422a45d99")
```
This ensure that the patch contains the full hash in the index lines, not the short hash which can change when additional commits are added to avoid short hash collisions.